build-system/eslint-rules/no-module-exports.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
   create(context) {
 
     function isModuleExports(node) {
```

Comment:
We can actually clean this up by using ESLint's selector interface:

```js
module.exports = {
  meta: {
    type: "suggestion",
    schema: [],
    messages: {
      unexpected: "Unexpected use of `module.exports` statement."
    }
  },

  create(context) {
    return {
      "MemberExpression[object.name=module][property.name=exports]"(node) {
        context.report(node);
      }
    };
  },
};

Reply:
Got it thanks :)